### PR TITLE
Cleanup api_guides/

### PR DESCRIPTION
**Pull request by lamberta**
Start cleaning out the api_guides.

First round removes guides that don't don't any guidance. These just list functions, which should be unnecessary in TF 2.0.

### CODE REVIEW

**Comment by lamberta** [comment]
Here's what's left: https://github.com/lamberta/docs/tree/remove-api_guides/site/en/api_guides/python

Want to look around for other easy pickings I can get in this CL?

File: site/en/api_guides/python/index.md [hunk 1]
```diff
@@ -1,52 +0,0 @@
-# Python API Guides
```
Comment (MarkDaoust) [inline site/en/api_guides/python/index.md]: This is an index file, not a list of ops... might want to keep, or add a redirect.

Comment (lamberta) [reply site/en/api_guides/python/index.md]: I'll add a redirect to /guide
It doesn't look like you can access this page from: https://www.tensorflow.org/api_docs/python/tf ?

Comment (lamberta) [reply site/en/api_guides/python/index.md]: cl/214344585

**Pull request merged**